libraries/AP_Camera/AP_Camera_Logging.cpp

```diff
@@ -41,6 +42,16 @@ void AP_Camera_Backend::Write_CameraInfo(enum LogMessages msg, uint64_t timestam
         altitude_gps = 0;
     }
 
+    float_t mount_roll;
+    float_t mount_pitch;
+    float_t mount_yaw;
+    auto *s = AP_Mount::get_singleton();
+    if(!s->get_attitude_euler(s->get_primary_instance(),mount_roll,mount_pitch,mount_yaw)) {
```

Comment:
Instead of getting the primary instance here, we could add a new "get_attitude_euler" in AP_Mount.h.  That would fit the regular pattern.